parso/python/tokenize.py

```diff
@@ -131,10 +131,10 @@ def _create_token_collection(version_info):
     Name = r'\w+'
 
     if version_info >= (3, 6):
-        Hexnumber = r'0[xX](?:_?[0-9a-fA-F])+'
-        Binnumber = r'0[bB](?:_?[01])+'
-        Octnumber = r'0[oO](?:_?[0-7])+'
-        Decnumber = r'(?:0(?:_?0)*|[1-9](?:_?[0-9])*)'
+        Hexnumber = r'0[xX](?:_?[0-9a-fA-F])+[lL]?'
+        Binnumber = r'0[bB](?:_?[01])+[lL]?'
+        Octnumber = r'0[oO](?:_?[0-7])+[lL]?'
+        Decnumber = r'(?:0(?:_?0)*[lL]?|[1-9](?:_?[0-9])*[lL]?)'
```

Comment:
This is Python 3.6+, definitely not needed, because this feature doesn't exist in Python 3.